src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx

```diff
@@ -190,15 +232,18 @@ describe('Accordion', () => {
         );
 
         expect(
-          wrapper.find(AccordionItemButton).find('div').props()['aria-expanded']
-        ).toEqual(false);
+          container
+            .querySelector('.accordionButtonDefault')
+            ?.closest('div')
+            ?.getAttribute('aria-expanded')
+        ).toEqual('false');
       });
     });
 
     describe('onChange prop', () => {
       it('is invoked with an array of expanded items’ uuids, if there are any', async () => {
```

Comment:
Any reason for this function to be async anymore?

Reply:
No it doesn't have to be async. I've removed it